vendor.mod

```diff
@@ -191,6 +193,7 @@ require (
 	github.com/spdx/tools-golang v0.5.3 // indirect
 	github.com/stretchr/testify v1.8.4 // indirect
 	github.com/syndtr/gocapability v0.0.0-20200815063812-42c35b437635 // indirect
+	github.com/thales-e-security/pool v0.0.2 // indirect
```

Comment:
This looks to be some un-blessed fork of https://github.com/vitessio/vitess; last commit is 4 Years ago, and last code-commit 5 Years ago 😬 https://github.com/thales-e-security/pool
<img width="869" alt="Screenshot 2024-05-30 at 11 26 43" src="https://github.com/moby/moby/assets/1804568/1bc26b3f-89b4-4494-a1bb-20b11ee499e8">